Adyen/Service/Resource/PosTerminalManagement/AssignTerminals.cs

```diff
@@ -0,0 +1,36 @@
+﻿#region Licence
+
+// 
+//                        ######
+//                        ######
+//  ############    ####( ######  #####. ######  ############   ############
+//  #############  #####( ######  #####. ######  #############  #############
+//         ######  #####( ######  #####. ######  #####  ######  #####  ######
+//  ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+//  ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  #############  #############  #############  #############  #####  ######
+//   ############   ############  #############   ############  #####  ######
+//                                       ######
+//                                #############
+//                                ############
+// 
+//  Adyen Dotnet API Library
+// 
+//  Copyright (c) 2020 Adyen B.V.
+//  This file is open source and available under the MIT license.
+//  See the LICENSE file for more info.
+
+#endregion
+
+using Adyen.Constants;
+
+namespace Adyen.Service.Resource.PosTerminalManagement
+{
+    public class AssignTerminals : ServiceResource
+    {
+        public AssignTerminals(AbstractService abstractService)
+            : base(abstractService, abstractService.Client.Config.PosTerminalManagementEndpoint + "/" + ClientConfig.PosTerminalManagementVersion + "/assignTerminals", null)
```

Comment:
Should we use the required fields?
```suggestion
            : base(abstractService, abstractService.Client.Config.PosTerminalManagementEndpoint + "/" + ClientConfig.PosTerminalManagementVersion + "/assignTerminals", new List<string>
                {
                    "companyAccount",
                    "terminals"
                })
```

Reply:
I added them 👍